### PR TITLE
Added Insights plug-in release note for 4.7

### PR DESCRIPTION
**Pull request by ahardin-rh**
@mmuehlfeldRH PTAL

### CODE REVIEW

**Comment by ahardin-rh** [comment]
Once https://github.com/openshift/openshift-docs/pull/26740 is merged, we can add a link to that content too.

**Comment by ahardin-rh** [comment]
@mmuehlfeldRH PTAL

**Comment by mmuehlfeldRH** [comment]
@ahardin-rh , the only thing I would change is:
...to learn more about the issues and associated risks.   ->   ...for further details about the issues and, if available, how to fix them.

Otherwise, LGTM.

**Comment by ahardin-rh** [comment]
@mmuehlfeldRH Updated. Thanks! I also added a link to your content.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -72,6 +72,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-web-console"]
 === Web console
 
+[id="ocp-4-7-web-console-insights-plugin"]
+==== Insights plug-in
+
+The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc#displaying-the-insights-status-in-the-web-console_using-insights-to-identify-issues-with-your-cluster[Insights plug-in] is now integrated into the {product-title} web console. Insights provides cluster health data, such as the number of total issues and total risks of the issues. Risks are labeled as *Critical*, *Important*, *Moderate*, or *Low* and you can quickly navigate to {cloud-redhat-com} for further details about the issues and, if available, how to fix them.
```
Comment (bobfuru) [inline release_notes/ocp-4-7-release-notes.adoc:78]: The last sentence could be broken up and the last part needs to be slightly reworded, perhaps just by removing "if available". Something like:
> Risks are labeled as *Critical*, *Important*, *Moderate*, or *Low*. You can quickly navigate to {cloud-redhat-com} for further details about the found issues and how to fix them.

**Review by bobfuru — COMMENTED** [review]
One suggestions and otherwise LGTM!

**Comment by ahardin-rh** [comment]
Updated! Merging. Thank you!

**Pull request merged**